src/main/interface/msp.c

```diff
@@ -768,10 +768,12 @@ bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst) {
     case MSP_NAME: {
         // Show warning for DJI OSD instead of pilot name
         // works if osd warnings enabled, osd_warn_dji is on and usb is not connected
```

Comment:
It no longer cares if USB connected. suggest change comment to // works if osd warnings enabled, and osd_warn_dji is on